Tests/test_seq.py

```diff
@@ -750,6 +751,7 @@ def test_set_wobble_codon_to_n(self):
 
 class TestUnknownSeq(unittest.TestCase):
     def setUp(self):
+        warnings.simplefilter("ignore", BiopythonDeprecationWarning)
```

Comment:
What scope does this have? i.e. side effects after this class has been tested?

Reply:
Good point. The filter needs to be restored to its default setting in `tearDown`.